### ISSUE TITLE
Eqn 3.7: Missing words in statements

### ISSUE THREAD

**Issue by wei2912** [learnyou/lysa#10]
Substituting `\not A` for `A \implies \bot` was mentioned but not the equivalent for `B`.

**Comment by pharpend** [comment]
On Tue, Jul 07, 2015 at 08:43:56AM -0700, Ng Wei En wrote:

> Substituting `\not A` for `A \implies \bot` was mentioned but not the
> equivalent for `B`.

I meant that as a general thing. Perhaps I should substitute that with:

```
For all $X$, $\lnot X$ means that $X \implies \bot$. Thus, $\lnot A$
means that $A \implies \bot$, and $\lnot B$ means $B \implies \bot$.
```

Thoughts?

I'm not sure I mentioned before that `Not x = x -> _|_` (in Idris
notation).

**Comment by wei2912** [comment]
Looks fine to me.

**Comment by pharpend** [comment]
Fixed in 1aaae32
